src/tools.py

```diff
@@ -121,7 +121,8 @@ def _colorsToHex(r: int = 0, g: int = 0, b: int = 0) -> str:
     
     return str.lower(f"#{r:02X}{g:02X}{b:02X}")
 
-def _formatForSVG(char: str):
-    if char in SVGMaps:
-        char = SVGMaps.get(char)
-    return char
+def pFormatForSVG(char: str) -> str | None:
```

Comment:
this is wrong. The original function returns the input character if it is not inside the SVG_MAPS list, whereas this version would return None, making the `converters.ConvertToSVG()` output wrong results in line 42 of the file src/converters.py. This function should never return none.